recipes/boehmgc/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set name = "boehmgc" %}
+{% set version = "8.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://www.hboehm.info/gc/gc_source/gc-{{ version }}.tar.gz
+  sha256: 436a0ddc67b1ac0b0405b61a9675bca9e075c8156f4debd1d06f3a56c7cd289d
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
```

```suggestion
test:
  commands:
    - test -f ${PREFIX}/lib/libboehmgc${SHLIB_EXT}   # [unix]
    - if not exist %LIBRARY_LIB%\boehmgc.lib exit 1       # [win]
```

So you can test if the shared objects are available in the package, that is just an example, I don't know the name of those libs which this package generates

---

OK. I think it's just called `gc`.